### PR TITLE
:children_crossing: Improve the workflow to delete own profile picture

### PR DESCRIPTION
**Pull request by MrKrisKrisu**

### CODE REVIEW

File: app/Http/Controllers/API/v1/SettingsController.php [hunk 1]
```diff
@@ -162,14 +161,26 @@ public function updateSettings(Request $request): UserProfileSettingsResource|Js
...
 
+    /**
+     * Undocumented and unofficial API Endpoint
+     *
+     * @return JsonResponse
+     */
     public function deleteProfilePicture(): JsonResponse {
         if (BackendSettingsController::deleteProfilePicture(user: auth()->user())) {
-            return $this->sendResponse('', 204);
+            return $this->sendResponse(['message' => __('settings.profilePicture.deleted')]);
```
Comment (HerrLevin) [inline app/Http/Controllers/API/v1/SettingsController.php:171]: Http code 204 missing

**Review by HerrLevin — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/API/v1/SettingsController.php (hunk 1 above)
Comment (MrKrisKrisu) [reply app/Http/Controllers/API/v1/SettingsController.php:171]: changed to 200 because there is content now

Comment (MrKrisKrisu) [reply app/Http/Controllers/API/v1/SettingsController.php:171]: ( 200 = default = missing declaration )

Comment (MrKrisKrisu) [reply app/Http/Controllers/API/v1/SettingsController.php:171]: image

**Pull request merged**